.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         path: |
           sccache
           src/agent/target
-        key: agent-${{ runner.os }}-${{ hashFiles('src/agent/Cargo.lock') }}
+        key: agent-${{ runner.os }}-${{ hashFiles('src/agent/Cargo.lock') }}-2021-01-14-01
```

Comment:
Is there something important about this choice of date? Do we need to preserve it?

Reply:
It's not that we need to preserve it, but we need to differentiate it from previous cache keys to get the benefit.

Reply:
Unfortunately, there isn't an easy way to invalidate a cache entry.  If you've not used a cache entry in 7 days, then it will automatically get removed.  However, that's either read or write.  :/  There isn't an API to evict the cache early.